cmd/ddev/cmd/addon-get.go

```diff
@@ -349,6 +346,8 @@ func createManifestFile(app *ddevapp.DdevApp, addonName string, repository strin
 func init() {
 	AddonGetCmd.Flags().String("version", "", `Specify a particular version of add-on to install`)
 	AddonGetCmd.Flags().BoolP("verbose", "v", false, "Extended/verbose output")
+	AddonGetCmd.Flags().String("project", "", "Name of the project to install the add-on in")
+	_ = AddonGetCmd.RegisterFlagCompletionFunc("project", ddevapp.GetProjectNamesFunc("all", 0))
```

Comment:
Intentionally ignoring these errors - completion is just icing on top, we shouldn't completely break DDEV in the unlikely event something goes wrong with this.